client/src/styles/bundle.scss

```diff
@@ -0,0 +1,10 @@
+// Core variables
+@import 'variables';
```

Comment:
See if this does anything, can't see variables.scss in agencyextensions

Reply:
Pulls from silverstripe-admin module due to silverstripe/webpack-config

Reply:
Ah yep - this is pretty common in some other modules for pulling default variables for admin. Perhaps we should comment this import in our modules 🤔